docs/source/conf.py

```diff
@@ -72,13 +72,6 @@
 # The master toctree document.
 master_doc = "index"
 
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-#
-# This is also used if you do content translation via gettext catalogs.
-# Usually you set "language" from the command line for these cases.
-language = None
```

Comment:
`WARNING: Invalid configuration value found: 'language = None'. Update your configuration to a valid langauge code. Falling back to 'en' (English).`

Can we replace `None` with `en`?

Reply:
Yes, I think we can do that too. The current change is based on the following, and results will be identical in either case in my understanding. Do you want me to change?

- https://github.com/sphinx-doc/sphinx/pull/10481/files#diff-da24b355349b53e4c8047034767d008e4c2ce9ea6c8933a3acd50d0828f00528
- https://github.com/sphinx-doc/sphinx/issues/10474#issuecomment-1140389657

Reply:
Thank you for your explanation. In the above PR, the Sphinx developer recommended the removal, so the current change looks reasonable.

> So please remove the configuration from these projects.